packages/cli/package.json

```diff
@@ -49,8 +49,10 @@
     "@vercel/python": "3.1.33",
     "@vercel/redwood": "1.0.44",
     "@vercel/remix": "1.1.6",
+    "@vercel/routing-utils": "workspace:2.1.3",
     "@vercel/ruby": "1.3.49",
     "@vercel/static-build": "1.1.0",
+    "i": "^0.3.7",
```

Comment:
What's this?

Reply:
that looks like a mistake. i was running a lot of `pnpm i` commands to get everything building

Reply:
Ha, I've done this before :)